src/absil/ilwrite.fsi

```diff
@@ -18,6 +18,7 @@ type ILStrongNameSigner =
 type options =
  { ilg: ILGlobals
    pdbfile: string option
+   portable: bool
```

Comment:
document here if portable means CoreCLR or something else and maybe give a (not going to be outdated) summary of things that deal with this option?